tdrs-backend/celery_start.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# export REDIS_URI=rediss://xtJnXoZpeLNID24pQhpPb9HSr@master.prd-3fa9afec-b332-43c8-9a46-0aaa72c49d53.pw1qnn.usgw1.cache.amazonaws.com:6379
+echo starting celery
+celery -A tdpservice.settings worker -c 1 &
```

Comment:
you can increase the number of workers: my suggestion is 3, unfortunately there is no definitive answer or rule but we can definitely increase the number of workers: https://docs.celeryq.dev/en/stable/userguide/workers.html#concurrency

Reply:
@jtimpe how many servers do you think we should aim for?

Reply:
i think 3 is a good starting point, though there are a lot of options we can play with
https://docs.celeryq.dev/en/stable/userguide/workers.html

i recall somewhere that 1 worker per available cpu core is ideal, but i can't find a reference for that so i could be making it up :)

Reply:
I'll put in 3 for now and if things go boom, we can revert :)